test/test_mbjson.py

```diff
@@ -54,7 +54,8 @@
     "standardize_instruments": True,
     "release_ars": True,
     "preferred_release_countries": [],
-    "artist_locale": 'en'
+    "artist_locale": 'en',
+    "originaldate_use_recording": True
```

Comment:
Since Python allows it, we should keep a comma at the end of the last line, it makes diffs cleaner.

Reply:
Actually this option was removed, so this line is not needed at all. I added the comma anyway as a small syntax change, we usually keep trailing commas in those cases. I would be even ok if we would test for this in our linting